app/follow-list/page.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import Loading from "@/common/Loading";
+import FollowList from "@/component/FollowList";
+import UserInfo from "@/component/UserInfo";
+import useGetFollowInfo from "@/libs/hook/useGetFollowInfo";
+import useGetUserInfo from "@/libs/hook/useGetUserInfo";
+import React from "react";
+
+const page = () => {
+  const { isLoading: userInfoLoading, data: userInfoData } = useGetUserInfo();
+  const { isLoading: followInfoLoading, data: followInfoData } =
+    useGetFollowInfo();
+
+  return userInfoLoading || followInfoLoading ? (
```

Comment:
isLoading이라는 것을 판단하는 변수를 만들어서 써주면 더 직관적일것 같아요!
```suggestion
const isLoading = useInfoLoading || followInfoLoading
  return isLoading ? (
```